app/src/main/java/org/listenbrainz/android/ui/screens/brainzplayer/BrainzPlayerBackDropScreen.kt

```diff
@@ -188,14 +189,19 @@ fun PlayerScreen(
             }
         }
         item {
-            Row(verticalAlignment = Alignment.CenterVertically) {
-                Column(modifier = Modifier.fillMaxWidth(0.8f)) {
+            Row(
+                verticalAlignment = Alignment.CenterVertically,
+                horizontalArrangement = Arrangement.SpaceBetween,
+                modifier = Modifier
+                    .padding(start = 25.dp, end = 25.dp)
```

Comment:
Please use `horizontal = ListenBrainzTheme.paddings.horizontal` as the argument here instead of `25.dp` so that we have consistent side padding across the app. Previously it was broken as well, so better fix it now while we are at it :)

Also, I do not recommend using `ListenBrainzTheme.paddings` everywhere. Sometimes, hardcoded paddings are the way, so that is that. Feel free to explore what paddings are there in the provider itself and think if any options feel like a fit, if not, hardcode :)